apps/etterlatte-vilkaarsvurdering/src/main/kotlin/vilkaarsvurdering/config/ApplicationContext.kt

```diff
@@ -20,6 +21,7 @@ class ApplicationContext {
     val vilkaarsvurderingService = VilkaarsvurderingService(
         vilkaarsvurderingRepository = VilkaarsvurderingRepository(dataSource),
         behandlingKlient = BehandlingKlientImpl(config, httpClient()),
-        grunnlagKlient = GrunnlagKlientImpl(config, httpClient())
+        grunnlagKlient = GrunnlagKlientImpl(config, httpClient()),
+        sessionFactory = PostgresSessionFactory(dataSource)
```

Comment:
Skulle vi laget denne i repository klassen i stedet for, slik at servicen slipper å ha to databaseting å forholde seg til? 
feks
```
class VilkaarsvurderingRepository(private val ds: DataSource) {
    val sessionFactory: SessionFactory
    init {
        sessionFactory = PostgresSessionFactory(ds)
    }
    fun createTransactionalSession(): SessionFactory {
        return this.sessionFactory
    }
```

Evt  flytte datasource ut av `VilkaarsvurderingRepository` og kun sende inn en session fra en tjeneste som tilbyr enten transactional eller "vanlig" ?